// @giphy/js-util/dist/index.js
"use strict";
function __export(m) {
    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
}
Object.defineProperty(exports, "__esModule", { value: true });
var bestfit_1 = require("./bestfit");
exports.bestfit = bestfit_1.default;
exports.setRenditionScaleUpMaxPixels = bestfit_1.setRenditionScaleUpMaxPixels;
__export(require("./collections"));
__export(require("./construct-moat-data"));
var get_client_rect_from_el_1 = require("./get-client-rect-from-el");
exports.getClientRect = get_client_rect_from_el_1.default;
var gif_utils_1 = require("./gif-utils");
exports.getAltText = gif_utils_1.getAltText;
exports.getBestRendition = gif_utils_1.getBestRendition;
exports.getBestRenditionUrl = gif_utils_1.getBestRenditionUrl;
exports.getGifHeight = gif_utils_1.getGifHeight;
exports.getGifWidth = gif_utils_1.getGifWidth;
exports.getSpecificRendition = gif_utils_1.getSpecificRendition;
__export(require("./log"));
__export(require("./sdk-headers"));
var tracking_pixel_1 = require("./tracking-pixel");
exports.injectTrackingPixel = tracking_pixel_1.default;
var webp_check_1 = require("./webp-check");
exports.checkIfWebP = webp_check_1.checkIfWebP;
