// @giphy/react-components/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var js_util_1 = require("@giphy/js-util");
require("intersection-observer");
var attribution_1 = require("./components/attribution");
Object.defineProperty(exports, "Attribution", { enumerable: true, get: function () { return attribution_1.default; } });
var overlay_1 = require("./components/attribution/overlay");
Object.defineProperty(exports, "AttributionOverlay", { enumerable: true, get: function () { return overlay_1.default; } });
var carousel_1 = require("./components/carousel");
Object.defineProperty(exports, "Carousel", { enumerable: true, get: function () { return carousel_1.default; } });
var gif_1 = require("./components/gif");
Object.defineProperty(exports, "Gif", { enumerable: true, get: function () { return gif_1.default; } });
Object.defineProperty(exports, "PingbackContext", { enumerable: true, get: function () { return gif_1.PingbackContext; } });
var grid_1 = require("./components/grid");
Object.defineProperty(exports, "Grid", { enumerable: true, get: function () { return grid_1.default; } });
var legacy_grid_1 = require("./components/legacy-grid");
Object.defineProperty(exports, "LegacyGrid", { enumerable: true, get: function () { return legacy_grid_1.default; } });
var version = require('../package.json').version;
// send headers with library type and version
js_util_1.appendGiphySDKRequestHeader("X-GIPHY-SDK-NAME", 'ReactSDK');
js_util_1.appendGiphySDKRequestHeader("X-GIPHY-SDK-VERSION", version);
